src/parse-stack-path.test.ts

```diff
@@ -0,0 +1,35 @@
+import { parseStackPath } from "./parse-stack-path";
+
+const EXAMPLE = [
```

Comment:
This is YAML?

Reply:
Yes, `stack path` and `stack query` output yaml.